test/order_test.rb

```diff
@@ -142,20 +142,36 @@
 
     it "Returns accurate information about the last order" do
       # TODO: Your test code here!
+      last = Order.all.last
+      puts last.inspect
+
+      expect(last.id).must_equal 100
+      expect(last.products).must_equal({"Amaranth" => 83.81, "Smoked Trout" => 70.6, "Cheddar" => 5.63})
+      expect(last.customer.id).must_equal 20
+      expect(last.fulfillment_status).must_equal :pending
     end
   end
 
   describe "Order.find" do
     it "Can find the first order from the CSV" do
       # TODO: Your test code here!
+      first = Order.find(1)
```

Comment:
Consider storing `1` in a variable such as `first_id` to increase readability and changeability.